src/infrastructure/persistance/ai_explanation_access.cpp

```diff
@@ -51,8 +57,8 @@ void AiExplanationAccess::getExplanation(const QString& authToken,
                 {
                     if(word[i] == '\n' && word[k] == '\n')
                     {
-                        word.removeAt(k);
-                        word.removeAt(i);
+                        word.remove(k,1);
+                        word.remove(i,1);
```

Comment:
In fact, this is for compatibility with older Qt versions, because most of popular Linux destroy like debian etc. don't have qt6.5 in their official repos. For example, this method will not compile with qt6. 4

Reply:
Ah I understand. I appreciate the care, but Librum uses multiple 6.5-only features, so this isn't important since Librum can only be build with 6.5+ anyways.

Reply:
Ok, I'll delete it, no problem.